openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/EntityRepository.java

```diff
@@ -2486,6 +2475,35 @@ private ChangeEvent getChangeEvent(
         .withPreviousVersion(prevVersion);
   }
 
+  protected void createAndInsertChangeEvent(
```

Comment:
Where is this used? I could not find a call.

Reply:
removed it.